lib/fetch.js

```diff
@@ -43,7 +43,6 @@ exports.fetchTransactions = async function() {
       name,
       date,
       amount: -amount,
-      category,
```

Comment:
Another option to add `category` to list of the map arguments on  line 41

Reply:
See #17